CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
  All notable changes to this project will be documented in this file.
  The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### [1.1.1] - 2019-03-26
```

Comment:
What's the reason of releasing this? Wouldn't be enough to keep those changes unreleased? There's no code change except of one component renaming.

Reply:
That makes sense to me. I will remove